test/e2e_node/seccompdefault_test.go

```diff
@@ -30,6 +30,7 @@ import (
 	kubeletconfig "k8s.io/kubernetes/pkg/kubelet/apis/config"
 	"k8s.io/kubernetes/test/e2e/framework"
 	e2eoutput "k8s.io/kubernetes/test/e2e/framework/pod/output"
+	"k8s.io/kubernetes/test/e2e/node"
```

Comment:
This import appears to add 120 new tests to the node E2E tests, many of which are failing.  

See https://testgrid.k8s.io/sig-node-release-blocking#node-kubelet-serial-containerd

Reply:
oh, talking about unexpected...

Reply:
I found out this was the problem commit and it took several minutes to figure out why it was causing a problem 😆

Reply:
Oh, do we need an additional CI check to prevent such imports?

Reply:
would be great. @dims @pohly is there anything similar to this already in k/k codebase to take example from?

Reply:
There is import boss. You could create a `test/e2e_node/.import-restrictions` file and add an entry for `k8s.io/kubernetes/test/e2e/.*`.

Reply:
thank you! @dims already helped with this. So many cool features in the codebase

Reply:
Thank you folks for the support here!